policy_packs/aws/backup/enforce_ebs_volume_backups/README.md

```diff
@@ -0,0 +1,183 @@
+>
+> Please note your Terraform state file will then become out of sync and the policy settings should then only be managed in the console.
+
+By default, the policies are set to `Check` in the pack's policy settings. To enable automated enforcements, you can switch these policies settings by adding a comment to the `Check` setting and removing the comment from one of the listed enforcement options:
+
+```hcl
+# AWS > Backup > Stack
+resource "turbot_policy_setting" "aws_backup_stack" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/aws-backup#/policy/types/backupStack"
+  #value    = "Check: Configured"
+  value    = "Enforce: Configured"
+}
+
+# AWS > IAM > Stack > Source
+resource "turbot_policy_setting" "aws_iam_iam_stack" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/aws-iam#/policy/types/iamStack"
+  # value    = "Check: Configured"
+  value    = "Enforce: Configured"
+}
+```
+
+Then re-apply the changes:
+
+```sh
+terraform plan
+terraform apply
+```
+
+## Disable Backups
+- To stop backups from happening while preserving the existing backups, remove this resource definition from the `AWS > Backup > Stack > Source` policy setting. With no resource
+```hcl
+    resource "aws_backup_selection" "ebs_resource_assignment" {
+      iam_role_arn = "arn:aws:iam::{{ $.account.id }}:role/turbot/core/guardrails_backup_service_role"
+      name         = "guardrails-ebs-resource-assignment"
+      plan_id      = aws_backup_plan.guardrails_ebs_backups.id
+      resources = ["arn:aws:ec2:*:*:volume/*"]
+    }
+```
+- Commit the updated policy setting via Terraform then let the `AWS > Backup > Stack` make the changes. 
+
+
+## Decommission the Backup Vault
+To completely remove a Backup Vault and all backups, execute the following steps:
+
+- Edit the Backup > Stack > Source policy setting in the policy pack.
+  - To flush the current backups, set the retention period to one day, as shown below. 
+  - Remove the Backup Selection from the `AWS > Backup > Stack > Source`.  This will prevent new backups from happening.
+
+When the changes are complete the `template` attribute for `Backup > Stack > Source` should look like this. 
+```hcl
+    |
+    resource "aws_backup_vault" "vault" {
+      name = "guardrails-backup-vault"
+      tags = {
+        turbot_version = "v5"
+      }
+    }
+    resource "aws_backup_plan" "guardrails_ebs_backups" {
+      name = "guardrails-backup-plan"
+      rule {
+        # The time allowed for the job to start, any longer and it will be cancelled.
+        start_window = 480
+        # The amount of time allowed for the backup to complete, before it is cancelled.
+        completion_window = 10080
+        #
+        rule_name         = "guardrails-ebs-backups-rule"
+        schedule          = "cron(0 5 ? * * *)"
+        target_vault_name = aws_backup_vault.vault.name
+        lifecycle {
+          delete_after = 1
+        }
+      }
+    }
+```
+
```

Comment:
It feels a bit redundant, but I think we should include the `terraform plan/terraform apply` instruction and code block like we have above so it's clear when users need to re-run the Terraform